File: RMS(Server)/RMS(Server)/Models/SummaryOfInventory.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace RMS_Server_.Models
{
    public class SummaryOfInventory
    {
        public int Id { get; set; }
        public string ItemUsedName { get; set; }
        public double ItemTotal { get; set; }
        public double ItemUsedToday { get; set; }
        public double ItemRemaining { get; set; }
        public Inventory Inventory { get; set; }
        public string InventoryId { get; set; }
        public int Unit { get; set; }
    }
}

File: RestaurantManagementApp/src/app/shared/summary-of-inventory.ts
import {Inventory} from './inventory.model';

export class SummaryOfInventory {

  public Id : number;
  public ItemUsedName: string;
  public ItemTotal: number;
  public ItemUsedToday: number;
  public ItemRemaining : number;
  public Inventory: Inventory[];
  public InventoryId : string;
  public Unit: number;

constructor(id: number, itemUsedName: string, itemTotal: number, itemUsedToday: number,
            itemRemaining: number, inventory: Inventory[], inventoryId: string, unit: number) {
    this.Id = id;
    this.ItemUsedName = itemUsedName;
    this.ItemTotal = itemTotal;
    this.ItemRemaining = itemRemaining;
    this.ItemUsedToday = itemUsedToday;
    this.Inventory = inventory;
    this.InventoryId = inventoryId;
    this.Unit = unit;
}

}
